.circleci/config.yml

```diff
@@ -387,6 +387,12 @@ jobs:
     steps:
       - test-linux:
           llvm: "10"
+  test-llvm10-go115:
+    docker:
+      - image: circleci/golang:1.15-buster
```

Comment:
We might want to stop testing some older versions eventually, such as 1.12 and 1.14 (keeping 1.11 and 1.13 in place as they had larger changes). That should speed up CI a bit and avoid unnecessary testing.

... but probably not in this PR.